tenable/io/v3/vm/permissions/api.py

```diff
@@ -44,13 +44,13 @@ def change(self, otype: str, id: UUID, *acls: Dict) -> None:
                 `permissions documentation`_ for more details.
 
                 i.e {'type': 'user', 'id': 2236706, 'permissions': 64}
-                    type:
-                        The type of permission (default, user, group).
-                    id (int):
-                        The unique ID of the user or group.
-                    permissions (int):
-                        The permission value to grant access as described in
-                        `permissions documentation`_.
+                type:
```

Comment:
Specify type of **type** param. Eg. **type (str)**.

Reply:
Updated

Reply:
Verified!